lms/envs/common.py

```diff
@@ -1470,7 +1474,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
 ################################# Middleware ###################################
 
 MIDDLEWARE = [
-    'openedx.core.lib.x_forwarded_for.middleware.XForwardedForMiddleware',
```

Comment:
Is this functionality duplicated in `django_cookies_samesite`?
I think this middleware is important for determining if requests are via HTTPS when forwarded through a proxy.
Is this something fixed in the updated gunicorn (20.0.4)? The middleware notes that it's a workaround for a breaking change in gunicorn 19.0. If so, should we remove the middleware entirely?

Reply:
Thanks for noticing that @kaizoku !  I don't think this should be removed, and definitely not as part of this ticket.  Have re-instated it, and will rebuild the sandbox.